dlpy/model_conversion/write_sas_code.py

```diff
@@ -67,27 +85,27 @@ def write_input_layer(model_name='sas', layer_name='data', channels='-1',
         '       elif (len(inputImageSize) == 3):',
         '           height,width = input_image_size[1:]',
         '       else:',
-        '           raise ValueError("input_image_size must be a tuple with two or three entries")',
+        '           raise ValueError("Parameter input_image_size must be a tuple with two or three entries")',
         '',
         '   # instantiate model',
         '   s.buildModel(model=dict(name=' + repr(model_name) + ',replace=True),type="CNN")',
         '',
         '   # input layer',
         '   nchannels=' + channels,
         '   if input_channel_offset is None and nchannels==3:',
-        '       print("INFO: setting channel mean values to ImageNet means")',
+        '       print("INFO: Setting channel mean values to ImageNet means")',
         '       input_channel_offset = [103.939, 116.779, 123.68]',
         '       s.addLayer(model=' + repr(model_name) + ', name=' + repr(layer_name) + ',',
         '                  layer=dict( type="input", nchannels=' + channels + ', width=' + width + ', height=' + height + ',',
-        '                  scale = ' + scale + ', randomcrop=input_crop_type, offsets=input_channel_offset))',
+        '                  scale = ' + scale + ', randomcrop=input_crop_type, offsets=input_channel_offset, offsetStd=norm_std))',
```

Comment:
I'm guessing that offsetStd is a parameter to addLayer and that's why it isn't underscore-delimited?

Reply:
That is correct.